wbooks/App.test.tsx

```diff
@@ -8,6 +8,16 @@ import renderer from 'react-test-renderer';
 
 import App from './src/app';
 
+const reactotron = {
+  configure: () => reactotron,
+  use: () => reactotron,
+  connect: () => reactotron,
+  createEnhancer: () => reactotron,
+  setAsyncStorageHandler: () => reactotron
+};
+
+jest.mock('reactotron-react-native', () => reactotron);
```

Comment:
We don't use this for mocks. Instead, create a `reactotron-react-native` file inside `__mocks_` folder and then
```
export default {
  configure: () => reactotron,
  use: () => reactotron,
  connect: () => reactotron,
  createEnhancer: () => reactotron,
  setAsyncStorageHandler: () => reactotron
}
```
But, is it really necessary?

Reply:
If I do not mock reactotron, the test fail

Reply:
Done!